apis/coursebatchmanapi/coursebatchmanapi.yaml

```diff
@@ -56,7 +56,843 @@ paths:
         required: true
         type: string
         description: >-
-            Type of the request data. Set as application/json.
+            The Content Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+      - name: ts
+        in: header
+        required: true
+        type: string
+        description: 'The time stamp at which the **create batch** request was sent. '
+      - name: X-msgid
+        in: header
+        required: true
+        type: string
+        description: 'It is a Unique ID that identifies the request incase the same API is executed multiple times.'
+      - name: x-authenticated-user-token
+        in: header
+        required: true
+        type: string
+        description: 'The alphanumeric string to access the API'
+      - name: Authorization
+        in: header
+        required: true
+        type: string
+        description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: 'SUCCESS. The **create batch** operation was successful!'
+          schema:
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/CreateBatchResponse'
+        '400':
+          description:  'BAD REQUEST,**create batch** operation failed !The possible reason for failure is that you may have missed providing input for a mandatory parameter' 
+          schema:  
+            allOf:
+             - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+             - $ref: '#/definitions/BatchCreateErrorResponse'
+        '404':
+          description: 'RESOURCE NOT FOUND ! **Create Batch**operation failed ! The possible reason for failure is that the provided course Id is invalid.'

```

Comment:
RESOURCE NOT FOUND! **Create Batch**operation failed! 
Possible failure reason: Entered invalid Course ID